1.2 Inventory.feature

```diff
@@ -0,0 +1,43 @@
+Feature: 1.2 Inventory
+
+Background: 
+Given main page URL is opened
```

Comment:
Which one? Sometimes url might be changed and your colleagues need to know which url should be used here